MsGraphicsPkg/Library/SimpleUIToolKit/ListBox.c

```diff
@@ -497,7 +497,7 @@ Draw (
 
     // If the ListBox was created with the checkbox option flag, toggle the checkbox select state.
     //
-    if ((L' ' == pKey->Key.UnicodeChar) && (UIT_LISTBOX_FLAGS_CHECKBOX == (this->m_Flags & UIT_LISTBOX_FLAGS_CHECKBOX))) {
+    if ((L' ' == pKey->Key.UnicodeChar)) {

```

Comment:
You can clean up an extra set of parentheses here.

Reply:
Sounds good.  Done.